cmake/std/atdm/ats2/environment.sh

```diff
@@ -186,8 +186,6 @@ if [[ "$ATDM_CONFIG_COMPILER" == "CUDA-10.1.243_"* ]]; then
   export KOKKOS_NUM_DEVICES=4
 
   # CTEST Settings
-  # TPETRA_ASSUME_CUDA_AWARE_MPI is used by cmake/std/atdm/ats2/trilinos_jsrun
-  export TPETRA_ASSUME_CUDA_AWARE_MPI=0
```

Comment:
Doesn't local-driver.sh run after environment.sh? Why is this export of `TPETRA_ASSUME_CUDA_AWARE_MPI` not picked up in local-driver?

Reply:
@e10harvey, the ctets-s-driver.sh script sources the load-env.sh script again and overwrites this.  We need to just not touch the `TPETRA_ASSUME_CUDA_AWARE_MPI` var in the atdm/environment.sh script.